pkg/jsonmessage/jsonmessage.go

```diff
@@ -124,6 +131,7 @@ func (jm *JSONMessage) Display(out io.Writer, isTerminal bool) error {
 	return nil
 }
 
+// DisplayJSONMessagesStream display json message stream.
```

Comment:
please be a bit more specific with what it does with `in` and `out` and `isTerminal`